### ISSUE TITLE
Feature: rule to match tag-name to class

### ISSUE THREAD

**Issue by hugo-vrijswijk** [43081j/eslint-plugin-lit#215]
Similar to [eslint-plugin-wc tag-name-matches-class](https://github.com/43081j/eslint-plugin-wc/blob/HEAD/docs/rules/tag-name-matches-class.md) a similar rule that matches on `@customElement`

Incorrect example:

```ts
@customElement("my-element")
export class FooBar extends LitElement {
```

Correct example:

```ts
@customElement("foo-bar")
export class FooBar extends LitElement {
```

Similar to the eslint-plugin-wc rule, a prefix and suffix option would be good

**Comment by 43081j** [comment]
Makes sense I think

Though it seems we need literally the same rule but looking at the decorator string rather than the define call

I wonder if we can somehow share the implementation. Maybe the WC plugin could somehow expose how it finds tag names so we can extend it instead of providing a separate rule

Maybe I'm overthinking though and we should just duplicate it 😅
